symbols.d.ts

```diff
@@ -0,0 +1,4 @@
+export const info: string;
+export const success: string;
+export const warning: string;
+export const error: string;
```

Comment:
I don't see the need to have these in a separate file. Can just be in index.d.ts.

Reply:
revert it!